src/cadence/scripts/v2/emulator/getNFTIDs.cdc

```diff
@@ -0,0 +1,5 @@
+import AlchemyMetadataWrapperEmulator from "../../../contracts/AlchemyMetadataWrapperEmulator.cdc"
```

Comment:
Any reason for the duplication of these scripts for each stack?

For testnet, I see we have staging vs production, but can that be flow-json controlled by account instead of per-script file?

Reply:
Ahh yeah we can make it 3 scripts, 1 per "stack" and at the account level we can toggle via the flow-json.  Ill update that